docs-app/.prettierignore

```diff
@@ -23,3 +23,7 @@
 /package.json.ember-try
 /package-lock.json.ember-try
 /yarn.lock.ember-try
+
+# ember-template-imports
+/**/*.gjs
+/**/*.gts
```

Comment:
Prettier support for `*.{gjs,gts}` files isn't great at the moment. For simplicity, we won't enforce formatting for now.